src/main/java/ai/timefold/solver/benchmarks/examples/tsp/domain/Visit.java

```diff
@@ -50,13 +63,29 @@ public void setPreviousStandstill(Standstill previousStandstill) {
      * @return a positive number, the distance multiplied by 1000 to avoid floating point arithmetic rounding errors
      */
     @JsonIgnore
+    @Override
     public long getDistanceFromPreviousStandstill() {
         if (previousStandstill == null) {
             return 0L;
         }
         return getDistanceFrom(previousStandstill);
     }
 
+    /**
+     * @return a positive number, the distance multiplied by 1000 to avoid floating point arithmetic rounding errors
+     */
+    @JsonIgnore
+    @Override
+    public long getDistanceToNextStandstill() {
```

Comment:
Do we actually need both next and previous? How about we standardize on one of them only?